posts/2020-07-30-json-logging-open-liberty-20008.adoc

```diff
@@ -0,0 +1,282 @@
+// Above:
+// Replace RELEASE_VERSION_PATH with the path in the following format: 
+//   issues?q=label%3Arelease%3A20003+label%3A%22release+bug%22+ 
+//   where `20003` is replaced with release version without the periods, eg: 20004
+// Replace the RELEASE_VERSION with the Open Liberty release number e.g. 20.0.0.4
+// // // // // // // //
+// tag::run[]
+[#run]
+// // // // // // // //
+// LINKS
+//
+// OpenLiberty.io site links:
+// link:{url-prefix}/guides/maven-intro.html[Maven]
+// 
+// Off-site links:
+//link:https://openapi-generator.tech/docs/installation#jar[Download Instructions]
+//
+// IMAGES
+//
+// Place images in ./img/blog/
+// Use the syntax:
+// image::/img/blog/log4j-rhocp-diagrams/current-problem.png[Logging problem diagram,width=70%,align="center"]
+// // // // // // // //
+== Run your apps using 20.0.0.8
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>20.0.0.8</version>
+    <type>zip</type>
+</dependency>
+----
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[20.0.0.8,)'
+}
+----
+Or if you're using Docker:
+[source]
+----
+FROM open-liberty
+----
+//end::run[]
+// // // // // // // //
+// Above:
+// Replace the RELEASE_VERSION with the Open Liberty release number e.g. 20.0.0.4
+// // // // // // // //
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+//tag::features[]
+[#accessJSON]
+=== HTTP access log fields in JSON logs
+// Add the introduction to the feature and description here
+In Open Liberty, you have the option to format your server logs in either basic or JSON format. When logs are in JSON format, you have to specify the sources (`message`, `trace`, `accessLog`, `ffdc`, `audit`) you want to send to `messages.log` or `console.log`/`standard-out`. This feature adds an option for you to include fields from the `accessLogging logFormat` attribute in the JSON logs. Previously, only selected fields would be printed in JSON logs. You are now able to include other NCSA access log fields in their JSON logs.
+
+Also, you now have the possibility to be able to define which JSON access log fields you want from the `accessLogging logFormat` property, which can then be sent to a log analysis tool, such as the ELK (Elasticsearch, Logstash, Kibana) stack. With this new feature, you can receive more informative logs that suit your needs. For example, you are able to specify that you want the user ID and request time fields in your JSON access logs -- then, these can be filtered by user ID in Kibana and track performance on a user-by-user basis.
```

Comment:
"property" to "attribute" to be consistent with her earlier usage.

Reply:
Done